Automattic/newspack-app-shell
# Enqueue scripts only if persistent element is present

## What it does

Prevents scripts from being loaded if no persistent element is present.

Closes #5 

## How to test

1. create the persistent element
1. visit the front-end - `client.js` script should be loaded and the App Shell navigation active
1. trash the persistent element post
1. visit the front-end - `client.js` script should not be loaded, navigation should reload the page like on a plain old website
